tomopyui/backend/io.py

```diff
@@ -102,7 +102,7 @@ def __getattr__(self, name):
         return object.__getattribute__(self, name)
 
     def save_normalized_as_npy(self):
-        np.save(self.filedir / "normalized_projections.npy", self.data)
+        np.save(os.path.join(self.filedir, "normalized_projections.npy"), self.data)
```

Comment:
Any specific reason for this?

Reply:
It was throwing an error at some point (TypeError: unsupported operand type(s) for /: 'str' and 'str') but I just switched it back and it actually works fine. I don't quite understand how that concatenation operator works actually, but I'll remove it from this merge

Reply:
self.filedir is a pathlib.Path instance : https://docs.python.org/3/library/pathlib.html. you can concatenate with "/" . if it is a string that follows it, should concatenate nicely. if the issue crops up again, we may have a string instead of a path.Pathlib instance for self.filedir.